agents-audit/pom.xml

```diff
@@ -421,5 +398,11 @@
             <version>${slf4j.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.code.findbugs</groupId>
+            <artifactId>jsr305</artifactId>
+            <version>${jsr305.version}</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
This is for `import javax.annotation.concurrent.ThreadSafe;` in `AmazonCloudWatchAuditDestination.java`.